src/pages/landing/ebooks/components/_hero.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import GetEbook from './_get-ebook'
+import { Flex , Box } from 'components/containers'
+import { Header, QueryImage, Text } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+
+const MainWrapper = styled(Box)`
+    background-image: ${(props) => props.bg};
+
+    @media ${device.tablet} {
+        background-image: ${(props) => props.bgMobile};
+    }
+`
+const HeaderBody = styled(Flex)`
+    @media ${device.tablet} {
+        flex-direction: column;
+    }
+`
+
+const TopHeaderImgWrapper = styled(Flex)`
+    @media ${device.tablet} {
+        order: 2;
+    }
+`
+const HeaderImage = styled(QueryImage)`
+    margin: auto;
+    padding: 15px;
+`
+
+const ContentWrapper = styled(Flex)`
+    float: right;
+
+    @media ${device.tablet} {
+        float: none;
+    }
+`
+
+const Content = styled(Box)`
+    @media ${device.tablet} {
+        float: none;
+    }
+`
+const SubTitle = styled(Header)`
+    @media screen and (min-width: 1200px) {
+        max-width: 600px;
+    }
+    @media ${device.laptopL} {
+        max-width: 400px;
+    }
+    @media ${device.tablet} {
+        max-width: 300px;
+    }
+`
```

Comment:
Why are we setting max widths on our `Header` component and not the parent wrapper yeah?